base/src/main/java/io/spine/code/proto/RejectionDeclaration.java

```diff
@@ -55,7 +55,7 @@
     }
 
     /**
-     * Returns {@code true} if the class name ends with {@code “Rejections”},
+     * Returns {@code true} if the class name ends with {@code "Rejections"},
```

Comment:
How this item and the similar changes are related to the essence of this PR? Let's discuss in person.